c10/CMakeLists.txt

```diff
@@ -0,0 +1,50 @@
+project(c10 LANGUAGES CXX)
```

Comment:
Add C here too, it's safer if you end up doing C feature tests

Reply:
Really you can just make this

```
project(c10 CXX C)
```
That will be consistent with the other projects